packages/lesswrong/components/tagging/SidebarSubtagsBox.tsx

```diff
@@ -82,15 +82,21 @@ const SidebarSubtagsBox = ({ tagId, className, classes }: { tagId; className?: s
         <h2>Posts in this space are about</h2>
       </ContentStyles>
       <span className={classes.root}>
+        <FooterTag
+          key={tag._id}
+          tag={tag}
+          hideScore={true}
+          popperCard={<WrappedTagPreview tag={tag} showRelatedTags={false} />}
+        />
```

Comment:
Include the tag itself in "Posts in this space are about". Clicking on it takes you to the wiki tab, which I think is fine

Reply:
Minor bug that probably doesn't need to block merging but should be fixed at some point: the tooltip for the tags goes off the right side of the screen.